packages/http-client-csharp/emitter/src/code-model-writer.ts

```diff
@@ -9,6 +9,13 @@ import { CSharpEmitterContext } from "./sdk-context.js";
 import { CodeModel } from "./type/code-model.js";
 import { Configuration } from "./type/configuration.js";
 
+/**
+ * Writes the code model to the output folder. Should only be used by autorest.csharp.
+ * @param context - The CSharp emitter context
+ * @param codeModel - The code model to write
+ * @param outputFolder - The output folder to write the code model to
+ * @beta
```

Comment:
I tried to mark this as Internal but api-extractor seems to not support it even though it suggests that as one of the tags.

Reply:
if you mark as `@internal` the typescript compiler should strip the type and not expose it at all

Reply:
Ah ok, so we do still need to export it, so I guess that is where the error comes from:
```
ERROR: Internal Error: Unable to analyze the export "writeCodeModel" in
C:/Users/jolov/repos/typespec/packages/http-client-csharp/dist/emitter/src/code-model-writer.d.ts

You have encountered a software defect. Please consider reporting the issue to the maintainers of this application.
```

Reply:
It is meant to be "Internal" in that it is only being exposed for one specific consumer that we own, but not "Internal" to the actual package, so I think we have to use "Beta".